src/UI/Radio.elm

```diff
@@ -151,6 +158,23 @@ withWidth width (RadioGroup prop opt) =
     RadioGroup prop { opt | width = width }
 
 
+{-| `Radio.withDisposition` determines whether the radio group's items are arranged horizontally or vertically.
+
+    Radio.withDisposition Radio.horizontalDisposition someRadioGroup
+
+-}
+withDisposition : Disposition -> RadioGroup id msg -> RadioGroup id msg
+withDisposition disposition (RadioGroup prop opt) =
+    RadioGroup prop { opt | disposition = disposition }
+
+
+{-| When displaying, arrange the buttons in horizontal lines.
+-}
+horizontalDisposition : Disposition
+horizontalDisposition =
+    Horizontal
+
```

Comment:
I'd call this just `horizontal` for short, since it's the only thing using the context in the module